test-specifications/crypto-service/accountCreateTransaction.md

```diff
@@ -293,13 +293,13 @@ https://docs.hedera.com/hedera/sdks-and-apis/rest-api
-| 6       | Creates an account with the staked node ID set to an invalid node ID                  | key=<VALID_KEY>, stakedNodeId=-100                                                           | The account creation fails with an INVALID_STAKING_ID response code from the network.                 | N                 |
-| 7       | Creates an account with a staked account ID and a staked node ID                      | key=<VALID_KEY>, stakedAccountId=<OPERATOR_ACCOUNT_ID>, stakedNodeId=<VALID_NETWORK_NODE_ID> | The account creation fails with an INVALID_STAKING_ID response code from the network.                 | N                 |
+| 1       | Creates an account with the staked account ID set to the operators account ID         | key=<VALID_KEY>, stakedAccountId=<OPERATOR_ACCOUNT_ID>                                       | The account creation succeeds and the account has a staking account ID equal to the input account ID. | Y                 |
+| 2       | Creates an account with the staked node ID set to a valid node ID                     | key=<VALID_KEY>, stakedNodeId=<VALID_NETWORK_NODE_ID>                                        | The account creation succeeds and the account has a staking node ID equal to the input node ID.       | Y                 |
+| 3       | Creates an account with the staked account ID set to an account ID that doesn't exist | key=<VALID_KEY>, stakedAccountId="123.456.789"                                               | The account creation fails with an INVALID_STAKING_ID response code from the network.                 | Y                 |
+| 4       | Creates an account with the staked node ID set to a node ID that doesn't exist        | key=<VALID_KEY>, stakedNodeId=123456789                                                      | The account creation fails with an INVALID_STAKING_ID response code from the network.                 | Y                 |
+| 5       | Creates an account with the staked account ID set to an empty account ID              | key=<VALID_KEY>, stakedAccountId=""                                                          | The account creation fails with and SDK internal error.                                               | Y                 |
+| 6       | Creates an account with the staked node ID set to an invalid node ID                  | key=<VALID_KEY>, stakedNodeId=-100                                                           | The account creation fails with an INVALID_STAKING_ID response code from the network.                 | Y                 |
+| 7       | Creates an account with a staked account ID and a staked node ID                      | key=<VALID_KEY>, stakedAccountId=<OPERATOR_ACCOUNT_ID>, stakedNodeId=<VALID_NETWORK_NODE_ID> | The account creation fails with an INVALID_STAKING_ID response code from the network.                 | Y                 |
```

Comment:
Actually noticed an issue with this test (at least for C++ SDK). Since both staked account ID and node ID are being set, this test (which actually should pass) is dependent on the order that the protobuf fields are set in the SDK. For example, if the SDK when building the transactions or SDK server sets the staked node ID first and then the staked account ID, the created account is staked to the input staked account ID. However if the staked account ID is set first, the created account is staked to the input staked node ID. Is there/should there be an enforced order to these for the SDKs?